HW_1/task2.py

```diff
@@ -0,0 +1,19 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа
+# сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#    60 -> 10  40  10
+S = int(input("Введите общее количество журавликов: "))
+# x + x + (x + x)* 2 = S
+# 6x = S
+# x = S // 6 - Сережа
+if S > 0 and S % 6 == 0:
+    k = S // 6 * 4
+    c = S // 6
+    p = c
```

Comment:
Наверное, проще было бы сделать так.
```suggestion
    p = c = S // 6
    k = p * 4
```